testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/MyResourcesTest.java

```diff
@@ -30,18 +30,20 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 import static java.util.Collections.singletonList;
 import static org.hamcrest.core.StringEndsWith.endsWith;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 import static org.keycloak.representations.idm.CredentialRepresentation.PASSWORD;
 import static org.keycloak.testsuite.auth.page.AuthRealm.TEST;
 import static org.keycloak.testsuite.util.WaitUtils.pause;
 import static org.keycloak.testsuite.util.WaitUtils.waitForModalFadeIn;
 import static org.keycloak.testsuite.util.WaitUtils.waitForModalFadeOut;
 
-public class MyResourcesTest extends AbstractAccountTest {
+public class MyResourcesTest extends BaseAccountPageTest {
```

Comment:
Please add some simple notifications checks to all test scenarios, similarly to e.g. [DeviceActivityTest](https://github.com/keycloak/keycloak/blob/de8ba75399209121f4238aef567795c1c4e7c938/testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/DeviceActivityTest.java#L158).